database/migrations/2021_01_21_175205_create_notification_reads_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateNotificationReadsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notification_reads', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('notification_id');
+            $table->unsignedInteger('user_id');
+            $table->timestamps();
+
+            // Foreign keys
+            $table->foreign('notification_id')
+                ->references('id')
+                ->on('notifications')
+                ->onDelete('cascade');
```

Comment:
If you want - there's a method called `cascadeOnDelete` that does this :)

Reply:
62eaa78